libbeat/docs/config-file-format.asciidoc

```diff
@@ -0,0 +1,355 @@
+[[config-file-format]]
+== Config file format
+
+Beats config files are based on http://www.yaml.org[YAML], a file format that is
+easier to read and write then other common data formats like XML or JSON.
+
+In beats all YAML files start with a "dictionary" (often called a "hash"). In
```

Comment:
@urso We need to be careful about introducing new terms to describe an existing technology. I don't see "object" in the YAML spec. Is there a precedent for using this term? If we're talking about a YAML file, we should use industry-standard terminology to describe it. I wonder if you could just start by explaining that YAML is composed of nested data structures (or collections?) and then go on to describe what the structures (or collections) can contain. I mean...your current description is already too restrictive because the collection might be a list rather than name/value pairs, right?